recipes/atpbar/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "atpbar" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aadeb599271ae545ef1adc0af2aacc3177d655806a8011f0a986fdb4171e0d49
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+  run_constrained:
+    - jupyter
+    - ipywidgets
+    - widgetsnbextension
+
+test:
+  imports:
+    - atpbar
+
+about:
+  home: https://github.com/alphatwirl/atpbar
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
According to the terms of the licence a copy of the text should be included the tarball. You can either include it in the recipe directory or make a new release having added `include LICENSE` to your `MANIFEST.in` file.

Reply:
And it's passing! 🎉 (you can ignore `win_64` on Azure, see #7715)

Do you want to make a new release that includes the licence file in the PyPI tarball or shall I merge?

Reply:
Thank you. I inlucded `LICENSE` in the receipe direcotry. I will include it in `MANIFEST.in` for the next release.

Reply:
In that case LGTM 👍